openedx/features/qverse_features/registration/signals.py

```diff
@@ -164,11 +168,17 @@ def _validate_single_csv_row(row):
     if len(row['mobile']) > MOBILE_NUMBER_MAX_LENGTH:
         errors.append('Mobile number is more than {} characters long.'.format(MOBILE_NUMBER_MAX_LENGTH))
 
-    if int(row['levelid']) < 1 or int(row['levelid']) > MAX_LEVEL_CHOICES:
-        errors.append('Level ID must be greater than 0 and smaller than {}.'.format(MAX_LEVEL_CHOICES+1))
+    try:
+        if int(row['levelid']) < 1 or int(row['levelid']) > MAX_LEVEL_CHOICES:
```

Comment:
nit: I guess, we can use range here too.

Reply:
done.

Reply:
range function will create a list of numbers smallest to largest so using that will take more processing here.